core/designsystem/src/main/java/co/kr/tnt/designsystem/component/Button.kt

```diff
@@ -0,0 +1,202 @@
+
+    Button(
+        onClick = onClick,
+        enabled = enabled,
+        colors = config.colors,
+        shape = RoundedCornerShape(config.cornerRadius),
+        border = BorderStroke(width = config.stroke, color = config.borderColor),
+        contentPadding = config.contentPadding,
+        modifier = modifier
+            .height(config.height)
+            .defaultMinSize(minWidth = 1.dp, minHeight = 1.dp),
+    ) {
+        Text(
+            text = text,
+            style = config.textStyle,
+        )
+    }
+}
+
+@Composable
+fun TnTIconButton(
+    size: ButtonSize = ButtonSize.Medium,
+    type: ButtonType = ButtonType.GrayOutline,
+    iconPosition: IconPosition,
+    text: String,
+    enabled: Boolean = true,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val config = getButtonConfig(size, type, enabled)
+
+    Button(
+        onClick = onClick,
+        enabled = enabled,
+        colors = config.colors,
+        shape = RoundedCornerShape(config.cornerRadius),
+        border = BorderStroke(width = config.stroke, color = config.borderColor),
+        contentPadding = config.contentPadding,
+        modifier = modifier
+            .height(config.height)
+            .defaultMinSize(minWidth = 1.dp, minHeight = 1.dp),
+    ) {
+        when (iconPosition) {
+            is IconPosition.Leading -> {
+                iconPosition.icon()
+                Text(
+                    text = text,
+                    style = config.textStyle,
+                    modifier = Modifier.padding(start = 4.dp),
+                )
+            }
+
+            is IconPosition.Trailing -> {
+                Text(
+                    text = text,
+                    style = config.textStyle,
+                    modifier = Modifier.padding(end = 4.dp),
+                )
+                iconPosition.icon()
+            }
+        }
+    }
+}
+
+@Composable
+fun TnTBottomButton(
+    text: String,
+    enabled: Boolean = true,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Button(
+        onClick = onClick,
+        enabled = enabled,
+        shape = RectangleShape,
+        colors = ButtonDefaults.buttonColors(
+            containerColor = TnTTheme.colors.neutralColors.Neutral900,
+            contentColor = TnTTheme.colors.neutralColors.Neutral50,
+            disabledContainerColor = TnTTheme.colors.neutralColors.Neutral300,
+            disabledContentColor = TnTTheme.colors.neutralColors.Neutral50,
+        ),
+        contentPadding = PaddingValues(top = 20.dp),
+        modifier = modifier
+            .fillMaxWidth()
+            .height(100.dp),
+    ) {
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.Top,
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Text(
+                text = text,
+                style = TnTTheme.typography.h4,
+            )
+        }
```

Comment:
```suggestion
        Text(
            text = text,
            style = TnTTheme.typography.h4,
            modifier = Modifier.align(Alignment.Top)
        )
```

`Column` 안감싸고 `Text` 로만 하위 컴포저블 구현 가능할 것 같아요 !_!

Reply:
오매 수정했습니다 감사합니다!